app/lib/katello/resources/candlepin/consumer.rb

```diff
@@ -233,6 +233,42 @@ def update(url, client_cert, client_key, ca_file, attributes)
           ensure
             RestClient.proxy = ""
           end
+
+          def bind_entitlements(organization, pool: {})
+            self.organization = organization
+            JSON.parse(self['entitlements'].post(nil, params: RestClient::ParamsArray
+              .new(cp_request_params + pool.to_a)))
```

Comment:
do we actually need any of the data used by cp_request_params in the response?

Reply:
Nope, but I thought this would be a sensible set of params to show users about their entitlement in the task output.

Reply:
Here's some example output from the task:

```
{"response"=>
  [[{"id"=>"8cda03884701255728231e4a7ad54966",
     "pool"=>
      {"id"=>"8a85f98160c50f7c60c4fc0e0eea0411",
       "quantity"=>7500,
       "consumed"=>5212,
       "productName"=>"SKU",
       "productId"=>"ES1301909",
       "subscriptionId"=>"2384076"},
     "quantity"=>1}]]}
```